src/main/java/com/ning/http/client/providers/grizzly/AsyncHttpClientFilter.java

```diff
@@ -56,21 +56,21 @@
 /**
  * Grizzly higher level async HTTP client filter, that works as a bridge between
  * AHC and Grizzly HTTP APIs.
- * 
+ *
```

Comment:
I would avoid changing the tab convention in projects to improve legibility in commits (it is easier to detect changes)